selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/opendeclaration/Eclipse0121Test.java

```diff
@@ -47,7 +48,7 @@
   @Inject private ProjectExplorer projectExplorer;
   @Inject private CodenvyEditor editor;
   @Inject private TestProjectServiceClient testProjectServiceClient;
-  @Inject private BrowserLogsUtil browserLogsUtil;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
```

Comment:
it's better to avoid injection of WebDriver into the test. Lets inject webdriver in the same way as it was done in **SeleniumWebDriverHelper** class.